### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by jnm2**
Saw one while doing the other PR, went down the rabbit hole. :D

### CODE REVIEW

File: src/Cake.Common.Tests/Unit/Build/BitbucketPipelines/Data/BitbucketPipelinesRepositoryInfoTests.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void Should_Return_Correct_Value()
                 var result = info.Tag;
 
                 // Then
-                Assert.Equal("BitbucketPiplines", result);
+                Assert.Equal("BitbucketPipelines", result);
```
Comment (jnm2) [inline src/Cake.Common.Tests/Unit/Build/BitbucketPipelines/Data/BitbucketPipelinesRepositoryInfoTests.cs]: This one is actually a test that either doesn't run or is about to start failing?

**Comment by gep13** [comment]
@cake-build/cake-team I can't see any problems with getting this merged.  Anyone else have concerns?

**Comment by patriksvensson** [comment]
@gep13 Merge away ☺️

**Comment by gep13** [comment]
@jnm2 your changes have been merged, thanks for your contribution 👍

**Comment by patriksvensson** [comment]
Merged! Thank you for your contribution. Much appreciated! 👍

**Pull request merged**